src/components/LoanCards/ExpandableLoanCard/ExpandableLoanCard.vue

```diff
@@ -0,0 +1,181 @@
+				<kv-icon name="small-x" />
+			</button>
+		</div>
+		<loan-card-image
+			class="loan-card-image-component"
+			:loan-id="loan.id"
+			:name="loan.name"
+			:retina-image-url="loan.image.retina"
+			:standard-image-url="loan.image.default"
+			:is-visitor="isVisitor"
+			:is-favorite="isFavorite"
+			:loan-image-hash="loan.image.hash"
+			:disable-link="disableImageLink"
+
+			@track-loan-card-interaction="trackInteraction"
+			@favorite-toggled="toggleFavorite"
+		/>
+		<div>
+			<div class="expandable-loan-card-bottom">
+				<borrower-info-header-expandable
+					:country="loan.geocode.country.name"
+					:activity="loan.activity.name"
+					:name="loan.name"
+					:loan-id="loan.id"
+					:single-line="true"
+					class="list-loan-card-borrower-info-header"
+				/>
+				<fundraising-status
+					:amount-left="amountLeft"
+					:percent-raised="percentRaised"
+					:expiring-soon-message="expiringSoonMessage"
+					:is-funded="loan.status==='funded'"
+					:single-line="true"
+				/>
+				<borrower-info-body
+					:amount="loan.loanAmount"
+					:borrower-count="loan.borrowerCount"
+					:name="loan.name"
+					:status="loan.status"
+					:use="loan.use"
+					:loan-id="loan.id"
+				/>
+				<action-button
+					class="expandable-loan-card-action-button"
+					:loan-id="loan.id"
+					:loan="loan"
+					:items-in-basket="itemsInBasket"
+					:is-lent-to="loan.userProperties.lentTo"
+					:is-funded="isFunded"
+					:is-selected-by-another="isSelectedByAnother"
+					:is-simple-lend-button="true"
+
+					@click.native="trackInteraction({
+						interactionType: 'addToBasket',
+						interactionElement: 'Lend25'
+					})"
+
+					@add-to-basket="$emit('add-to-basket', $event)"
+				/>
+				<matching-text
+					:matching-text="loan.matchingText"
+					:is-funded="isFunded"
+					:is-selected-by-another="isSelectedByAnother"
+				/>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import activeLoanMixin from '@/plugins/active-loan-mixin';
+import ActionButton from '@/components/LoanCards/Buttons/ActionButton';
+import BorrowerInfoHeaderExpandable from '@/components/LoanCards/BorrowerInfo/BorrowerInfoHeaderExpandable';
+import BorrowerInfoBody from '@/components/LoanCards/BorrowerInfo/BorrowerInfoBody';
+import FundraisingStatus from '@/components/LoanCards/FundraisingStatus';
+import LoanCardImage from '@/components/LoanCards/LoanCardImage';
+import MatchingText from '@/components/LoanCards/MatchingText';
+import expandableLoanCardMixin from '@/components/LoanCards/ExpandableLoanCard/expandableLoanCardMixin';
+import KvIcon from '@/components/Kv/KvIcon';
+
+export default {
+	components: {
+		ActionButton,
+		BorrowerInfoHeaderExpandable,
+		BorrowerInfoBody,
+		FundraisingStatus,
+		LoanCardImage,
+		MatchingText,
+		KvIcon,
+	},
+	mixins: [
+		expandableLoanCardMixin,
+		activeLoanMixin,
+	],
+	computed: {
+		disableImageLink() {
+			return this.usingTouch && !this.expanded;
```

Comment:
This is cool, when the card is collapsed on a touch device, we can now tap on the image to expand the card (desired functionality). When the card is expanded, tapping the image will go to the borrower profile